pkg/cmd/deletecmd/delete_application.go

```diff
@@ -139,6 +139,23 @@ func (o *DeleteApplicationOptions) Run() error {
 	return nil
 }
 
+func removeEnvironments(jobs []string, environments map[string]*v1.Environment) (applications []string) {
```

Comment:
I am not a friend of naked return values. If anything they should only be used in short functions (which this one arguably is). The problem is that functions can change and easily cross the threshold where naked returns become more of a liability than anything else. Also for consistency reasons, I'd prefer to stay clean of them. That's my 0.02$

Reply:
I've switched the function to explicitly returns the list of applications